### PR TITLE
Added !antighost command

### PR DESCRIPTION
**Pull request by ghost**
This allows anyone to send an !antighost command to have all the botbot bots in the room set their nick again.  Will allow users to fix all bots that have ghosted (referring to a bot who is now a lurker instead of a bot, but still responds to commands, NOT referring to bots who are visible on the nicklist but have stopped responding)

### CODE REVIEW

**Comment by ghost** [comment]
I don't know why travis ci is failing, never dealt with travis ci before.

File: source/botbotbot.py [hunk 1]
```diff
@@ -119,6 +119,9 @@ def recv_message(self, content='', parent=None, this_message=None, sender='', se
                 self.pause(pause_text='/me has been paused by "' + sender + '".', reply_to=this_message)
         elif self.paused and EuphUtils.command('!help', self.nickname).match(content):
             self.pause(reply_to=this_message)
+        elif content == '!antighost':
```
Comment (ArkaneMoose) [inline source/botbotbot.py]: Please change this line to `elif EuphUtils.command('!antighost').match(content)`. This makes it consistent with the lines around it and more flexible (e.g. matches case-insensitively, strips whitespace, etc.).

**Comment by ArkaneMoose** [comment]
Hello, and thanks so much for your contribution!

I tested your pull request, and it seems to work excellently. My only suggestion is to change one line (see [here](https://github.com/ArkaneMoose/BotBot/pull/39/files#r78444461)). After that, I'll be happy to merge it.

And don't worry about the Travis CI build failing. At one point, I wanted to start using it, but never actually set it up properly. So every build will fail.

File: source/botbotbot.py (hunk 1 above)
Comment (ghost) [reply source/botbotbot.py]: I looked into that, however it appeared that the EuphUtils.command() method checks to see if the @mention is in the command, so I decided to do it this way.  If you want I could write a method in EuphUtils that doesn't require the @mention to be present but does all the other checking.  If so let me know what the method should be called.

Comment (ghost) [reply source/botbotbot.py]: nevermind, I just reviewed it and I'm dumb

**Comment by ghost** [comment]
Excellent! I have made the requested change!

**Pull request merged**